tests/test_private_attributes.py

```diff
@@ -131,7 +131,13 @@ class Model(BaseModel):
     m._foo = None
     assert m._foo is None
 
-    with pytest.raises(ValueError, match='"Model" object has no field "_bar"'):
```

Comment:
This test was a hold-over from v1, where it checked that classvars did not produce fields. I think adding a clearer error message is an improvement though, and clearly it's not a challenge to do so, which, at least when this test was created, I suspect is related to why it wasn't a separate error message in v1.